src/TraceEvent/TraceEvent.Tests/EventPipeParsing.cs

```diff
@@ -254,6 +255,21 @@ public void V4EventPipeFileHasProcNumbers()
             }
         }
 
+        [Fact]
+        public void GotoWorksForPositionsGreaterThanAlignment()
+        {
+            using (var reader = new PinnedStreamReader(new MockHugeStream((long)uint.MaxValue + 5_000_000), config: new SerializationConfiguration { StreamReaderAlignment = StreamReaderAlignment.EightBytes }))
+            {
+                reader.Goto((StreamLabel)0x148);
+                var buf = new byte[100_000];
+                // Specifically doing a read larger than the default buffer size (16KB) to avoid caching path.
```

Comment:
nit: Do you want to fail the test if someone changes the default buffer size to be larger than the buffer length here so your assumption is not accidentally invalidated?

Reply:
Probably easier to specifically set the buffer size of the stream to be smaller/an explicit size. I'll update the test.